client/src/App.jsx

```diff
@@ -29,10 +31,15 @@ class App extends Component {
               exact
               path="/captains/profile/:id"
               render={() => (
-                <div>
+                <div className="app" >
```

Comment:
@Anies12 It might be better to use a different className than "app" here, because that className is already being used up above.

Reply:
thanks Tom, this file is actually being edited with every pull request from each one of us, that's why you can see a mess here :)

Reply:
😂 I think every App.js ever written has that same problem